src/InfoBarWidget.js

```diff
@@ -118,4 +120,18 @@ InfoBarWidget.prototype.setErrorMessage = function ( errCode = 'refresh' ) {
 	this.setLabel( new OO.ui.HtmlSnippet( mw.msg( 'ext-whowrotethat-state-error', errorMessage ) ) );
 };
 
+/**
+ * Show the given username information
+ *
+ * @param {string} username
+ */
+InfoBarWidget.prototype.setUsernameInfo = function ( username ) {
+	this.userInfoUsernameLabel.setLabel( mw.msg( 'ext-whowrotethat-revision-username', username ) );
+};
+
+// Clear the username information
+InfoBarWidget.prototype.clearUsernameInfo = function () {
+	this.userInfoUsernameLabel.setLabel( '' );
```

Comment:
We should look into the usability here; when the screen is narrower, and the username is longer, the info bar jumps to accommodate 2 lines instead of one. Flagging this as something we should check into when QA'ing and for product to decide.

Reply:
I showed this to Ilana; don't bother touching this in this PR, but it's a good-to-know, especially when we deal with languages like German where the sentences can be a lot longer.

We'll keep an eye on this in QA and see if we need to do some UX followup.

Reply:
Sounds good, I did notice it. We can add a placeholder to avoid the jump. We'll see what product decides :)